docs/tutorial/GetStarted.md

```diff
@@ -323,11 +327,109 @@ The service `name`, `cpus` and `mem` are populated from the default values in th
 
 If you need further examples, you can refer to the [repo/packages/H/hello-world](/repo/packages/H/hello-world) package.
 
+### Step 3.1 : DC/OS Integration
+
+By default, a DC/OS service is deployed on a [private agent node](https://dcos.io/docs/1.9/overview/concepts/#private-agent-node). To allow configuration control or monitoring of a service by a user, the Admin Router can act as a reverse proxy by proxying calls on the master node to the service in a private node on the cluster.
```

Comment:
"To allow a user to control configuration or monitor a service, use the admin router as a reverse proxy. Admin router can proxy calls on the master node to a service on a private node.